### DISCUSSION TITLE
Cancellation Policy Criteria's and Standardization

### DISCUSSION THREAD

**Discussion by 92shreyansh** [ONDC-Official/mobility-specification#41] [Ideas]
Hi All,

Lets use this thread to discuss and identify possible criteria's for cancellation policy and its calculation which would help the Network to streamline and make the values machine computable.

**Comment by sashokkumarsaravanan** [comment]
Dear ONDC Community,

We're excited to share that as part of our ongoing efforts to enhance user experience and streamline our services we're implementing a new set of cancellation policies for scheduled rides within the ONDC network. These policies are designed to provide clarity and fairness to both buyers and sellers and will be applicable across our taxi and fleet services in India.

Our recommendation for the community is to adopt the following cancellation policies, which have been carefully crafted to balance flexibility for users with operational feasibility for service providers:

FREE: Cancellations are free until 15 minutes post-booking creation. Post the start time, no refund is provided.

SUPER_FLEXI: Cancellations are free up to 1 hour before the start time. Within 1 hour of the start time, no refund is provided.

FLEXI: Free cancellation is available until 6 hours before the start time. Between 6 to 1 hour prior to the start time, a cancellation fee of 50% of the booking amount or Rs 1000 whichever is lower is charged. No refund is given within 1 hour of the start time.

NON_REFUNDABLE: Free cancellation is permitted until 30 minutes after booking creation. Beyond this timeframe, no refund is available.

REFUNDABLE: Free cancellation is allowed up until the start time of the booking. Thereafter, no refund is offered.

SUPER_FLEXI_AT: Free cancellation up to 1 hour before the start time. From 1 hour to the start time a 100% penalty of the booking amount or Rs 1000, whichever is lower is applied. No refund is given after the start time.

FLEXI_AT: Free cancellation up to 1 hour before the start time. From 1 hour before to the start time a 50% penalty of the booking amount or Rs 1000, whichever is lower is charged. Beyond this no refund is provided.

Please note that the rates policy names, and timings associated with these policies are samples and can be customized by the seller to suit their service model.

To ensure transparency and informed decision-making, we advise the buyer's app to display these cancellation policies to the users when they request cab/fleet details. This way, each ride request will be accompanied by the applicable cancellation policy, minimizing potential disputes and enhancing user satisfaction.

We believe in the collective intelligence of our community and welcome suggestions from all members. Your insights are valuable to us and could help us refine these policies further. Please feel free to share your thoughts and recommendations.

We look forward to your feedback and active participation in adopting these policies, which we believe will contribute to the progressive growth of the ONDC ecosystem.

Warm regards,

Ashokkumar Saravanan
On behalf of Taxida Private Limited 
ONDC Community Team.
